src/index.html

```diff
@@ -7,8 +7,36 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navbar">
+      <a href="#home" class="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li><a href="#" class="is-active nav__link">Apple</a></li>
+          <li><a href="#" class="nav__link">Samsung</a></li>
+          <li><a href="#" class="nav__link">Smartphones</a></li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+              >Laptops & Computers
+            </a>
```

Comment:
```suggestion
          <li>
            <a href="#" class="is-active nav__link">
              Apple
            </a>
          </li>
          
          <li>
            <a href="#" class="nav__link">
              Samsung
            </a>
          </li>
          
          <li>
            <a href="#" class="nav__link">
              Smartphones
            </a>
          </li>
          
          <li>
            <a
                href="#"
                class="nav__link"
                data-qa="hover"
              >
                Laptops & Computers
            </a>
```
fix code style all over the page